src/Models/UserModel.ts

```diff
@@ -0,0 +1,63 @@
+import { Schema, Document, model, Model } from "mongoose";
+
+export interface IUser {
+  firstname: string;
+  lastName: string;
+  fullName: string;
+  phoneNumbers?: string;
+  age?: number;
+  gender?: string;
+  email: string;
+  userDeleted?: boolean;
+  clerkId: string;
+}
+
+export interface IUserDocument extends IUser, Document {
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const UserSchema = new Schema<IUserDocument>(
+  {
+    clerkId: {
+      type: "string",
+      required: true,
+    },
+    firstname: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    fullName: {
+      type: String,
+      required: true,
+    },
+    phoneNumbers: {
+      type: Array,
+    },
+    age: {
+      type: Number,
+    },
+    gender: {
+      type: String,
+    },
+    email: {
```

Comment:
Add a validation logic for this field in order evaluate the correctness of the email pattern. You check out the validation property from the mongoose docs. That will allow you to include a check as well as a custom validation message for this field.